modules/ROOT/pages/authentication-authorization/sso-integration.adoc

```diff
@@ -185,6 +185,12 @@ Custom-defined roles, such as `rolename`, must be explicitly created using the `
 See xref:authentication-authorization/manage-roles.adoc[Manage roles].
 ====
 
+[NOTE]
+====
+When specifying explicit group to role mapping the automatic mapping for groups and roles sharing a name is disabled.
+This means that all groups and roles need to be specified to be mapped, even if they share a name.
```

Comment:
Question for docs reviewer: Do we want to also cherry-pick this back to 4.4 or do we think 5.x is enough?

Reply:
We still support 4.4. So, if this note is valid for 4.4, then yes.